### PR TITLE
CCL 2023 Ingestion

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

File: data/yaml/name_variants.yaml [hunk 1]
```diff
@@ -5460,6 +5460,9 @@
 - canonical: {first: Yang, last: Liu}
   comment: Edinburgh
   id: yang-liu-edinburgh
+- canonical: {first: Yang, last: Liu}
```
Comment (mjpost) [inline data/yaml/name_variants.yaml:5463]: Are we sure this is a new Yang Liu? If unsure, I would just use the name without an "id", for later sorting.

**Review by mjpost — APPROVED** [review]

File: data/yaml/name_variants.yaml (hunk 1 above)
Comment (mbollmann) [reply data/yaml/name_variants.yaml:5463]: That's not possible, is it? All instances of ambiguous names must be disambiguated, or we raise an error.

Comment (mjpost) [reply data/yaml/name_variants.yaml:5463]: Correct, thanks for clarifying. You'd have to use the ID of the "base class", e.g., `id="yang-liu".

Comment (anthology-assist) [reply data/yaml/name_variants.yaml:5463]: The institution associated with this author is new.

**Comment by anthology-assist** [comment]
@mjpost this year CCL also has a `findings` volume to be ingested. Do we want this volume with other findings or with CCL?

**Comment by mjpost** [comment]
CCL is not an ACL event, so it is not venue ID `findings`. So it should likely be `2023.ccl-findings` instead of `2023.findings-ccl`. Can you ask them what this volume is?

**Pull request merged**